.docker/cli.dockerfile

```diff
@@ -31,7 +31,7 @@ ENV GITHUB_TOKEN=${GITHUB_TOKEN}
 ENV WEBROOT=${WEBROOT} \
     COMPOSER_ALLOW_SUPERUSER=1 \
     COMPOSER_CACHE_DIR=/tmp/.composer/cache \
-    SIMPLETEST_DB=mysql://drupal:drupal@mariadb/drupal \
+    SIMPLETEST_DB=mysql://lagoon:lagoon@mariadb/lagoon \
```

Comment:
Please do not do it this way

Please update the docker image to sustain the existing drupal credentials

https://github.com/drevops/vortex/pull/1537/files#diff-a7dbd602e0c858c5292a079f029e46bafefcf920a564577361617237a4b51462R16

Reply:
@richardgaunt ^^

Reply:
Thanks Alex - good point